README.md

```diff
@@ -165,6 +168,11 @@ in the version tag.
 </dependency>
 ```
 
+### Using Other Dependency Management Configurations
+Please refer to the instructions available on [Maven Central](https://search.maven.org/artifact/com.cyberark.conjur.api/conjur-api/3.0.1/jar) 
```

Comment:
We probably don't want to link permanently to 3.0.1 - can we link [here](https://search.maven.org/artifact/com.cyberark.conjur.api/conjur-api) instead and just be clear that they need to click on a version to get the instructions?

Reply:
Great point! Thanks for that, I changed it to use that link and mention choosing a version.